src/lib/block/kuznyechik/kuznyechik.cpp

```diff
@@ -0,0 +1,4520 @@
+/*
+* Kuznyechik
+* This code is written by kerukuro for cppcrypto library (http://cppcrypto.sourceforge.net/)
+* and released into public domain.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/mem_ops.h>
+#include <botan/internal/kuznyechik.h>
+#include <botan/internal/loadstor.h>
+
+namespace Botan {
+
+namespace {
+
+namespace Kuznyechik_F {
+
+const unsigned char S[256] = {
```

Comment:
All integer types should be `uintX_t` (`uint8_t` here)

Reply:
Changed. Will be in the next update to this PR.